packages/js/product-editor/src/blocks/product-fields/name/editor.scss

```
@@ -30,7 +30,13 @@
 		}
 	}
 
-	.components-input-control__suffix {
-		margin-right: 0;
+	.components-input-control {
```

```suggestion
	.components-base-control .components-input-control__container {
		.components-input-control {
			&__input {
				font-size: $grid-unit-20;
				min-height: $grid-unit-60;
			}
			&__suffix {
				margin-right: 0;
			}
		}
	}
```

With this you don't need to use `!important` anymore.

---

Good call! Thank you @mdperez86  for the suggestion!